File: COM-1308/Operations Sheet/9/3.c
#include <stdio.h>

int main() {
   int x = 3, y = 10;

   if ((3 / x) < y) x = x + 2;
   y = y + 3;

   printf("x = %d\n", x);
   printf("y = %d\n", y);

   printf("\n");
   return 0;

   /* output:
   x = 5
   y = 13
   */
}

File: COM-1305/Exercise/7.2/3.py
import datetime

numbers = [int(num) for num in input("Enter Time (0 0 0) : ").split()[:3]]

if len(numbers) < 3:
    numbers += [0] * (3 - len(numbers))


def create_time(hours, minutes, seconds):
    time = datetime.time(hours, minutes, seconds)
    return time


print(f"Create Custom Time : {create_time(*numbers)}")

# NOTE: Input
# Enter Time (0 0 0) : 9 52 0

# NOTE: Output
# Create Custom Time : 09:52:00
